toyota_prius_2017_pt_generated.dbc

```diff
@@ -451,7 +451,7 @@ BO_ 608 STEER_TORQUE_SENSOR: 8 XXX
  SG_ STEER_TORQUE_DRIVER : 15|16@0- (1,0) [-32768|32767] "" XXX
  SG_ STEER_OVERRIDE : 0|1@0+ (1,0) [0|1] "" XXX
  SG_ CHECKSUM : 63|8@0+ (1,0) [0|255] "" XXX
- SG_ STEER_ANGLE : 31|16@0- (0.0573,0) [-500|500] "" XXX 
+ SG_ STEER_ANGLE : 31|16@0- (0.0573,0) [-500|500] "" XXX
```

Comment:
Is this change from an IDE?

Reply:
Yes, Atom

Reply:
Make sure it's reverted, you don't want to edit any of the generated files manually

Reply:
Derp, I didn’t run the generator